src/main/java/com/evanlennick/retry4j/config/RetryConfigBuilder.java

```diff
@@ -117,7 +118,23 @@ public final RetryConfigBuilder retryOnAnyExceptionExcluding(Class<? extends Exc
 
     public final RetryConfigBuilder retryOnReturnValue(Object value) {
         retryOnValue = true;
-        valueToRetryOn = value;
+        valuesToRetryOn = new Object[] { value };
+
+        return this;
+    }
+
+    @SafeVarargs
+    public final RetryConfigBuilder retryOnReturnValues(Object... values) {
+        retryOnValue = true;
```

Comment:
i would generally prefer wrapping these with a Collection of some sort rather than storing them as a raw array... for example: https://github.com/elennick/retry4j/pull/79/files#diff-25b653eb79d0323ac9625faf0ae65ef9R101

Reply:
Fair enough. Updated.